guides/common/modules/snip_managing-packages-api.adoc

```diff
@@ -0,0 +1,67 @@
+. Optional: If you want to {PackageAction} packages as a specific user, create an `ssh` object with the following fields of the string type:
+* `"effective_user"` with the name of the ssh user
+* `"effective_user_password"` with the password of the ssh user if this password is required
+. Optional: If you want to {PackageAction} packages at a later time, create the `"scheduling"` object.
+The object can contain one or both of the following fields of the string type with date, time, and a timezone in the ISO 8601 format:
+* `"start_at"`- sets the time to {PackageAction} the packages
+* `"start_before"` - sets the latest time to {PackageAction} the packages.
+If it is not possible to {PackageAction} the packages by this time, then this action is cancelled.
+
++
+If you omit time, it defaults to 00:00:00.
+If you omit timezone, it defaults to UTC.
+. Optional: If you want to limit the number of hosts on which the job is run concurrently, create the `"concurrency_control"` object with the `"concurrency_level"` field of the integer type.
+Assign the number of hosts as the field value.
+. Optional: If you want to {PackageAction} packages at a later time and you want the host search query to be evaluated at a time of running the job, create a `"targeting_type"` field of the string type with the `"dynamic_query"` value.
+This is useful if you expect the result of the search query to be different at the time of running the job due to changed status of the hosts.
+If you omit this field, it defaults to `"static_query"`.
+
+. Send a `POST` request with the created body to the `/api/job_invocations` endpoint of your {ProjectServer} and use a tool like python to see a formatted response.
++
+Example API request:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+curl https://_{foreman-example-com}_/api/job_invocations \
+-H "content-type: application/json" \
+-X POST \
+-d @_Path_To_My_API_Request_Body_ \
+-u _My_Username_:__My_Password__ | python3 -m json.tool
+----
+
+.Verification
+* In the {ProjectWebUI}, navigate to *Monitor* > *Jobs* and see the report of the scheduled or completed remote execution job to {PackageAction} the packages on the selected hosts.
+
+.An Example API Request Body
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{
+    "job_invocation" : {
+                     "inputs" : {
+                    "package" : "nano vim"
+        },
+                    "feature" : "katello_package_{PackageAction}",
+               "search_query" : "*",
+                        "ssh" : {
+             "effective_user" : "My_Username",
+    "effective_user_password" : "My_Password"
+        },
+                 "scheduling" : {
+                    "start_at": "2023-09-21T19:00:00+00:00",
+                "start_before": "2023-09-23T00:00:00+00:00"
+                 },
+        "concurrency_control" : {
+           "concurrency_level": 100
+        },
+             "targeting_type" : "dynamic_query"
+    }
+}
```

Comment:
To be honest I'm not a huge fan of this colon-aligned format, could we do two spaces per level?

```suggestion
{
   "job_invocation" : {
      "concurrency_control" : {
         "concurrency_level" : 100
      },
      "feature" : "katello_package_{PackageAction}",
      "inputs" : {
         "package" : "nano vim"
      },
      "scheduling" : {
         "start_at" : "2023-09-21T19:00:00+00:00",
         "start_before" : "2023-09-23T00:00:00+00:00"
      },
      "search_query" : "*",
      "ssh" : {
         "effective_user" : "My_Username",
         "effective_user_password" : "My_Password"
      },
      "targeting_type" : "dynamic_query"
   }
}
```

Reply:
For sure. To be honest, I just kept the format hammer threw at me :sweat_smile: